frame/clearing-house/src/lib.rs

```diff
@@ -8,6 +8,8 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+mod weights;
```

Comment:
i doubt i will be able to review this work until lib.rs or README.md added with documentation or reference to it.